jobsets/tests/simple-node-nixops.nix

```diff
@@ -0,0 +1,22 @@
+import ./make-test.nix ({ pkgs, ... }: {
+  name = "boot";
+  nodes = {
+    machine = { config, pkgs, ... }: {
+      imports = [ ../../modules/cardano-node.nix <nixops/nix/options.nix> <nixops/nix/resource.nix> ];
```

Comment:
that was to test the code surrounding networking.privateIPv4, because it both exists, and doesn't exist, depending on if i import nixops/nix/options.nix
because nothing tries to use the values of those options, it doesn't fail out